react/NestedSelect/Modal.jsx

```diff
@@ -1,21 +1,46 @@
 import React from 'react'
 import Icon from '../Icon'
 
-import { Img } from '../Media'
+import { Media, Bd, Img } from '../Media'
 import palette from '../palette'
 import NestedSelect from './NestedSelect'
 import Dialog, {
   DialogTitle,
   DialogContent
 } from 'cozy-ui/transpiled/react/Dialog'
+import IconButton from 'cozy-ui/transpiled/react/IconButton'
 import { useCozyDialog, DialogCloseButton } from '../CozyDialogs'
 
 import styles from './styles.styl'
 
 import LeftIcon from 'cozy-ui/transpiled/react/Icons/Left'
 
+const NestedSelectDialogHeader = ({ onClickBack, showBack, title }) => {
+  const { dialogTitleProps } = useCozyDialog({
+    open: true
+  })
+  return (
+    <DialogTitle {...dialogTitleProps}>
+      <Media>
+        {showBack && (
+          <Img className={styles.Modal__back}>
+            <IconButton onClick={onClickBack}>
+              <Icon icon={LeftIcon} color={palette['coolGrey']} />
```

Comment:
Est-ce que coolGrey peut-être remplacé par quelque chose de plus sémantique ?

Reply:
Good point, I was actually able to completely remove the color props since IconButton already are colored with `theme.palette.text.secondary`.